packages/rest-typings/src/index.ts

```diff
@@ -127,8 +131,10 @@ export type UrlParams<T extends string> = string extends T
 	: T extends `${infer _Start}:${infer Param}/${infer Rest}`
 	? { [k in Param | keyof UrlParams<Rest>]: string }
 	: T extends `${infer _Start}:${infer Param}`
+	? { [k in Param | keyof UrlParams<_Start>]: string }
```

Comment:
`_Start` is supposed to be ignored. What's the goal here? Parsing `:abc:def/rest`?